core/src/main/scala/dagr/core/util/ReflectiveBuilder.scala

```diff
@@ -26,7 +26,7 @@ class ReflectiveBuilder[T](val clazz: Class[T]) {
   protected val typeSignature = clazzSymbol.typeSignature
   protected val constructor   = findConstructor
   protected val jConstructor  = toJavaConstructor(constructor)
-  lazy val argumentLookup = new ArgumentLookup[ArgumentDescription]()
+  private[util] lazy val argumentLookup = new ArgumentLookup[ArgumentDescription]()
```

Comment:
I think this should remain at the same access level at the class - i.e. public.  Without it, the class is effectively useless.